src/TSHelper.ts

```diff
@@ -220,12 +222,50 @@ export class TSHelper {
         return [false, null];
     }
 
+    public static isNonCompoundPrefixUnaryOperator(node: ts.PrefixUnaryExpression): boolean {
```

Comment:
Would it not be nicer to have the opposite of this function and just use a negation where you need this? Depends on what the most common case is I guess.